flash/image/instance_segmentation/backbones.py

```diff
@@ -19,7 +19,7 @@
 from flash.core.integrations.icevision.backbones import get_backbones, load_icevision_ignore_image_size
 from flash.core.model import Task
 from flash.core.registry import FlashRegistry
-from flash.core.utilities.imports import _ICEVISION_AVAILABLE, _module_available, _TORCHVISION_AVAILABLE
+from flash.core.utilities.imports import _ICEVISION_AVAILABLE, _TORCHVISION_AVAILABLE, module_available
```

Comment:
```suggestion
from flash.core.utilities.imports import _ICEVISION_AVAILABLE, _TORCHVISION_AVAILABLE
from lightning_utilities.core.imports import module_available
```

Same reason as mentioned in the review comment above. :)